test/unit/org/apache/cassandra/db/compaction/unified/ControllerTest.java

```diff
@@ -121,6 +121,50 @@ public void testValidateOptionsIntegers()
         testValidateOptions(true);
     }
 
+    @Test
+    public void testValidateOptionsInvalidTargetSSTableSize()
+    {
+        //Test 1: TARGET_SSTABLE_SIZE_OPTION = 12E899, the value reported in CASSANDRA-20398 
+        try {
```

Comment:
Cassandra's style doesn't match std java style, `{` are on new lines by themself.  If you did that by hand, just keep in mind you need to do the cassandra style (see https://cassandra.apache.org/_/development/code_style.html).  If your idea did this, then you should rebuild your IDE configs from `ant`.

If you use Inteillij, then `ant generate-idea-files` will setup intelij to "do the right thing" most of the time for this project.

Reply:
done. Thanks